solara/util.py

```diff
@@ -346,3 +347,25 @@ def wrapper():
         return return_value
 
     return wrapper
+
+
+def deprecated(reason: str):
+    """
+    Mark functions as deprecated.
+    """
+
+    def decorator(func):
+        @functools.wraps(func)
+        def wrapped(*args, **kwargs):
+            warnings.simplefilter("always", DeprecationWarning)
```

Comment:
I don't think we should call this

Reply:
The reason this was originally included is that Python doesn't show `DeprecationWarning`s by default. However, it seems that it is recommended to use `FutureWarning` for warnings that you want users to see (as per [scikit learn](https://scikit-learn.org/0.22/whats_new/v0.22.html#deprecations-using-futurewarning-from-now-on)), so I made the warning category configurable by default instead.